### PR TITLE
[Mudah.my] New ruleset

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

File: src/chrome/content/rules/Mudah.my.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		- ooglewww	(t)
+		- panel	(m)
+		- stompinsabah	(t)
+		- survey	(t)
+		- ww	(t)
+		- www.ww	(t)
+		- ww43	(t)
+		- wwa	(t)
+		- www1	(t)
+		- www2	(t)
```
Comment (J0WI) [inline src/chrome/content/rules/Mudah.my.xml]: WFM

File: src/chrome/content/rules/Mudah.my.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		- www9	(t)
+		- wwww	(t)
+		- www.wwww	(t)
+		- wwwww	(t)
```
Comment (J0WI) [inline src/chrome/content/rules/Mudah.my.xml]: Wildcard DNS (all `(t)`)

File: src/chrome/content/rules/Mudah.my.xml (hunk 1 above)
Comment (youdly) [reply src/chrome/content/rules/Mudah.my.xml]: Added

File: src/chrome/content/rules/Mudah.my.xml (hunk 2 above)
Comment (J0WI) [reply src/chrome/content/rules/Mudah.my.xml]: A comment about the wildcard should be added.

Comment (youdly) [reply src/chrome/content/rules/Mudah.my.xml]: Done

**Pull request merged**